src/ripple/protocol/SField.h

```diff
@@ -150,7 +153,8 @@ class SField
         const char* fn,
         int meta = sMD_Default,
         IsSigning signing = IsSigning::yes);
-    explicit SField(private_access_tag_t, int fc);
+    explicit SField(private_access_tag_t, int fc) = delete;
+    explicit SField(private_access_tag_t, int fc, bool);
```

Comment:
The `bool` parameter in this signature is unused.  Remove the `bool`, which takes this signature back to the original signature.